src/tokenserver/migrations/2021-07-16-001122_init/up.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE `services` (
```

Comment:
I'll note that the services table may be able to be killed entirely.

Tokenserver has solely been used for one service, syncstorage. Either there's always been one entry in it with a pattern of "1.5" or possibly 2 entries (an additional one for pattern "1.1"). cc @rfk if he recalls this

It may make sense to keep it around for now to simplify things, particularly for the eventual migration, but FYI.

Reply:
> Either there's always been one entry in it with a pattern of "1.5" or possibly 2 entries (an additional one for pattern "1.1")

I believe there has only ever been a single entry in production use in this table; references to `1.1` are hangovers from testing and development.

Reply:
Just to be very clear: +100 to simplifying the `services` table out of existence entirely

Reply:
Cool -- I'll add a ticket to remove the `services` table as part of a separate ticket 👍🏻